test/framework/cloudstack.go

```diff
@@ -100,6 +102,16 @@ func UpdateRedhatTemplate124Var() api.CloudStackFiller {
 	return api.WithCloudStackStringFromEnvVar(cloudstackTemplateRedhat124Var, api.WithCloudStackTemplateForAllMachines)
 }
 
+// UpdateRedhatTemplate125Var updates the CloudStackTemplate for all machines to the one corresponding to K8s 1.24.
+func UpdateRedhatTemplate125Var() api.CloudStackFiller {
+	return api.WithCloudStackStringFromEnvVar(cloudstackTemplateRedhat125Var, api.WithCloudStackTemplateForAllMachines)
+}
+
+// UpdateRedhatTemplate126Var updates the CloudStackTemplate for all machines to the one corresponding to K8s 1.24.
+func UpdateRedhatTemplate126Var() api.CloudStackFiller {
+	return api.WithCloudStackStringFromEnvVar(cloudstackTemplateRedhat126Var, api.WithCloudStackTemplateForAllMachines)
+}
```

Comment:
Let's not rely on the env var for the template name. Instead use similar logic as in `WithCloudStackRedhat125` to infer the template name